runtime/crab/src/lib.rs

```diff
@@ -1321,18 +1321,18 @@ impl frame_support::traits::OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 		// --- substrate ---
 		use frame_support::migration::*;
 
-		if let Some(term) = take_storage_value::<darwinia_relay_primitives::relay_authorities::Term>(
+		put_storage_value::<([u8; 32], Vec<(AccountId, [u8; 65])>)>(
```

Comment:
Why we need this migration? @AurevoirXavier @wuminzhe

Reply:
Re-trigger *sign authorities*.

Reply:
https://github.com/darwinia-network/darwinia-common/blob/master/frame/bridge/relay-authorities/src/lib.rs#L492